newrelic-agent/src/main/java/com/newrelic/agent/transport/apache/ApacheSSLManager.java

```diff
@@ -36,12 +37,16 @@
     private static final Collection<String> NEW_RELIC_CERTS = ImmutableList.of("newrelic-com.pem",
             "eu-newrelic-com.pem", "eu01-nr-data-net.pem");
 
-    public static SSLContext createSSLContext(String caBundlePath) {
+    public static SSLContext createSSLContext(DataSenderConfig config) {
         SSLContextBuilder sslContextBuilder = new SSLContextBuilder();
         try {
-            if (caBundlePath != null) {
-                sslContextBuilder.loadTrustMaterial(getKeyStore(caBundlePath), null);
-            } else {
+            if (config.getCaBundlePath() != null) {
+                if (config.getUsePrivateSSL()) {
+                   Agent.LOG.log(Level.FINE, "Ignoring use_private_ssl config." +
```

Comment:
Shouldn't we log this at WARN or ERROR?

Reply:
I think it is probably ok to leave at `FINE`. It's not an error, or even problematic, condition. It really just indicates that one config option took precedent over another.